### PR TITLE
Remove probing - catalyst adds all the appropriate fields now

### PR DESCRIPTION
**Pull request by mjh1**

### CODE REVIEW

**Review by victorges — APPROVED** [review]
Niiiiice

File: task/upload.go [hunk 1]
```diff
@@ -480,19 +478,6 @@ func complementCatalystPipeline(tctx *TaskContext, assetSpec api.AssetSpec, call
 		}
 	}
 
-	if !FlagCatalystProbesFile {
```
Comment (victorges) [inline task/upload.go:483]: I'm almost sure there is one huge optimization we can do now: if catalyst copied the source file (`catalystCopiedSource`) and there is no IPFS storage (L481), we don't even need to download the source file anymore! Can probably do an early return around L447.

Haven't double checked alllll the logic that still remains here tho, so do make sure, but it seems overall possible 👀

**Comment by mjh1** [comment]
@victorges does the final logic look ok to you?

**Comment by victorges** [comment]
@mjh1 Sorry didnt see the comment til now. It does look good!

**Pull request merged**